elements/pfe-jump-links/src/pfe-jump-links-panel.js

```diff
@@ -142,14 +140,6 @@ class PfeJumpLinksPanel extends PFElement {
     this.sectionMargin = newVal;
   }
 
-  _isValidMarkup() {
```

Comment:
I was seeing this error on the pfe-accordion demo page but really it was just missed in the previous PR for jump links.  This rule is no longer in place (content can exist at any level inside the DOM).

Reply:
Can we move this into a separate issue?